db/migrate/20180221142221_migrate_registry_mirror_from_pillar.rb

```diff
@@ -0,0 +1,43 @@
+class MigrateRegistryMirrorFromPillar < ActiveRecord::Migration
+  def up
+    registry = Registry.create(
```

Comment:
This registry should only be created if the pillar previously existed, no?

Reply:
at least on v3 we depend on the suse registry entry to be present always, hence the mirror configuring possibility during bootstrap

Reply:
TBH I am a bit confused with this. I am OK with this `registry` entry being created from the very beginning, but I thought this was going to be created when on the UI we choose to mirror the SUSE registry on premise, instead of precreating this registry without knowing if this will be ever be used. I'm also thinking about kubic.

Reply:
AFIAK, user won't be able to do anything regarding the SUSE registry but only about its mirrors. So this might make sense but not sure about exporting it to the docker daemon.json.

Reply:
hmm good point there, should we skip it in the pillars controller when there's no mirror?

Reply:
@ereslibre updated to only create the registry on demand